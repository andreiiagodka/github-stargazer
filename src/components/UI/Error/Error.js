import React from 'react'

import { Alert } from 'react-bootstrap'

const Error = () => (
  <Alert variant='danger' className='text-center'>
    Something went wrong!
  </Alert>
)

export default Error