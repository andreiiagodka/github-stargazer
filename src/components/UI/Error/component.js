import React from 'react'

import { Alert } from 'react-bootstrap'

const ErrorComponent = () => (
  <Alert variant='danger' className='text-center'>
    Something went wrong!
  </Alert>
)

export default ErrorComponent