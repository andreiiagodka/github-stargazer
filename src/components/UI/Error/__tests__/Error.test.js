import React from 'react'
import renderer from 'react-test-renderer'

import Error from '../Error'

describe('<Error /> component', () => {
  it('renders component correctly', () => {
    const tree = renderer.create(<Error />).toJSON()
  
    expect(tree).toMatchSnapshot()
  })
})
