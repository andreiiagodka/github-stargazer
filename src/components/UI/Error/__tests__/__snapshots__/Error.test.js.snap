// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> component renders component correctly 1`] = `
<div
  className="fade text-center alert alert-danger show"
  role="alert"
>
  Something went wrong!
</div>
`;
